startmenu.py

```diff
@@ -66,7 +68,8 @@ def start_game(screen):
     """
     screen.addstr(0, 0, "Game will now start")
     screen.refresh()
-    time.sleep(3)
+
+    game_board.main(screen)
```

Comment:
You can remove the `start_game` function and just call `game_board.main` from the event loop directly

Reply:
fixed